src/app/services/range.py

```diff
@@ -5,6 +5,7 @@
 from app.helper.decorators import transaction_decorator
 from app.helper.errors import RangeNotExist
 from app.models import Range
+from app.models.range import RangeSchema
```

Comment:
You can import RangeSchema from app/models/__init__.py

Reply:
fixed